docusaurus/docs/reference/cli-commands.mdx

```diff
@@ -56,7 +57,7 @@ To learn more about the various types of plugins, refer to [Grafana plugin types
 
 For more information on how Scenes allows you to create dashboard-like experiences in app plugins, see the [Scenes](https://grafana.com/developers/scenes) documentation.
 
-### 2. Add a backend to support server-side functionality? (y/N)
+### Add a backend to support server-side functionality? (y/n)
```

Comment:
@josmperez the capital N is important as it indicates default behaviour - so that if you press enter then it assumes false

Reply:
Yeah, good point. I have changed it back in https://github.com/grafana/plugin-tools/pull/1042/commits/1602109407bdfd5d6ad5926f9809633e92287815.